content/code-style/laravel-php.md

```diff
@@ -75,6 +75,24 @@ Always use fully qualified class names in docblocks.
  */
 ```
 
+## Ternary operators
+
+Use multi-line ternay operators unless it's a really short expression.
+
+```php
+// Good
+
+$result = $flag && $object instanceof Model
```

Comment:
I don't like the fact that this example has `&&` in it. I know it's subjective, but I feel that it makes the example kinda "heavy".